thirdparty/git.go

```diff
@@ -271,6 +271,9 @@ func ParseGitUrl(url string) (string, string, error) {
 		url = strings.TrimPrefix(url, httpsPrefix)
 		split := strings.Split(url, "/")
 		if len(split) != 3 {
+			if split[0] == "api.github.com" && len(split) > 3 {
```

Comment:
Can we clarify that this is for the special case where we're using a GitHub API URL of the form `https://api.github.com/repos/<owner>/<repo>` rather than a git clone URL? Alternatively, since this function is meant to parse `git clone` URLs and the module URL is actually a GitHub API URL, consider making a separate function to parse GitHub API URLs.

Reply:
I think using the same function is fine because it's a generic "ParseGitUrl" function. I'll add a comment.